### PR TITLE
Instructions for using config and snippets library

### PR DESCRIPTION
**Pull request by aturner-epcc**
Addressing #112. Added instructions on building lessons that includes brief description of how to use config options and snippets. Would value comments on if we need a full description of what each config option is or whether we are happy for people to work it out by looking at the source.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,20 +11,44 @@ The lesson outline and rough breakdown of topics by lesson writer is in
...
+## Using this material
+
+1. Follow the instructions found in the [Software Carpentry example lesson source](https://github.com/carpentries/lesson-example/)
+   to create a repository for your lesson.
+2. Edit [_config.yml](_config.yml) to modify the configuration options for the HPC system you
+   will be using in the section titled `Workshop specific values`. These options set such things
+   as the address of the host to login to, definitions of the command prompt, scheduler names.
+   The default is the setup for the Graham Compute Canada cluster hosted at the University
+   of Waterloo which uses the SLURM scheduler. Other examples can be found in the
+   `_extras/snippets_library/` directory.
```
Comment (psteinb) [inline README.md]:
```suggestion
   [_extras/snippets_library/](_extras/snippets_library/) directory.
```

File: README.md [hunk 2]
```diff
@@ -22,20 +11,44 @@ The lesson outline and rough breakdown of topics by lesson writer is in
...
+   1. Code snippets are in files named `snippet_name.snip` and are included automatically
+      when the lesson is built. For example, if the `snippet_name` was `login_output`,
+      then the snippet file would be called `login_output.snip`.
+   2. Code snippets are placed in subdirectories that are named according to the episode they
+      appear in. For example, if the snippet is for episode 01, then it will be in a 
+      subdirectory called `01`.
+   3. The default is the setup for the Graham Compute Canada cluster hosted at the University
+      of Waterloo which uses the SLURM scheduler. Other examples can be found in the
+      `_extras/snippets_library/` directory.
+
```
Comment (psteinb) [inline README.md]: I think `3.` should be replaced by an example on how the snippet is included in the `_episodes` markdown. I think point 3. is more a comment than an entry in a numbered list.

Comment (aturner-epcc) [reply README.md]: Addressed in https://github.com/hpc-carpentry/hpc-intro/pull/113/commits/f4ca9d0689e31d11a86a77e7cc7671ce627e88da

**Comment by aturner-epcc** [comment]
@psteinb Thanks for the review and suggestions. I think I have addressed all you comments now. Can you take another look?

**Pull request merged**